### PR TITLE
Cook info setup 

### PR DESCRIPTION
**Pull request by noorafi**
cook info setup +meal card cook section

### CODE REVIEW

**Comment by Karyum** [comment]
@noorafi could you add the issue number in the description and assign your team members to review it

File: client/src/pages/MealPage/sections/MealInfo/MealInfo.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ class MealInfo extends Component {
     this.props.getMealInfo(this.props.mealId);
   }
   render() {
+    console.log(this.props);
```
Comment (matthall8) [inline client/src/pages/MealPage/sections/MealInfo/MealInfo.js]: No console logs in production code please!

File: server/controllers/cookInfo.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const { getCookInfo } = require('../database/getQueries/getCookInfo');
+
+exports.get = (req, res) => {
+  const { mealId } = req.params;
+  getCookInfo(mealId)
+    .then(row => res.send(row))
+    .catch(() => res.send('Could not find cook'));
```
Comment (matthall8) [inline server/controllers/cookInfo.js]: res.send should include res.status(500).

File: server/database/getQueries/mealReviews.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+const mealReviews = mealId => {
+  const queryString = `
+  SELECT users.username, reviews.star_rating, reviews.reviewed_at,
+   reviews.review_msg WHERE reviews.meal_id = $1;
+  `;
+  return db.one(queryString, [mealId]);
+};
+mealReviews(1)
+  .then(row => console.log(row))
```
Comment (matthall8) [inline server/database/getQueries/mealReviews.js]: No console log in the final code please.

File: server/views/error.hbs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<section class="error">
```
Comment (matthall8) [inline server/views/error.hbs]: No .hbs needed as discussed

File: client/src/pages/MealPage/sections/CookInfo/CookInfo.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    isFulfilled: PropTypes.bool,
+    isPending: PropTypes.bool,
+    error: PropTypes.string
+  })
+};
+const mapStateToProps = ({ cookData }) => ({
+  cookInfo: cookData
+});
+const mapDispatchToProps = {
+  getCookInfo
```
Comment (DenisKent) [inline client/src/pages/MealPage/sections/CookInfo/CookInfo.js]: Rename as mentioned above?

File: client/src/reducers/getCookInfoReducer.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
+import { GET_COOK_INFO } from "../actions/actionTypes";
```
Comment (DenisKent) [inline client/src/reducers/getCookInfoReducer.js]: name change as above

File: client/src/reducers/index.js [hunk 7]
```diff
@@ -2,9 +2,11 @@ import { combineReducers } from "redux";
...
 import getMealInfoReducer from "./getMealInfoReducer";
 import getMealListReducer from "./getMealListReducer";
+import getCookInfoReducer from "./getCookInfoReducer";
 
 export default combineReducers({
   addToBasket: addBasketReducer,
   mealData: getMealInfoReducer,
-  mealList: getMealListReducer
+  mealList: getMealListReducer,
+  cookData: getCookInfoReducer
```
Comment (DenisKent) [inline client/src/reducers/index.js]: name as above

File: server/controllers/cookInfo.js (hunk 2 above)
Comment (DenisKent) [reply server/controllers/cookInfo.js]: like so: res.status(500).res.send('Could not find cook')

File: server/controllers/index.js [hunk 8]
```diff
@@ -6,8 +6,11 @@ const router = express.Router();
 const mealInfo = require('./mealInfo');
 const mealList = require('./mealList');
 
+const cookInfo = require('./cookInfo');
```
Comment (DenisKent) [inline server/controllers/index.js]: name as above

File: server/controllers/index.js [hunk 9]
```diff
@@ -6,8 +6,11 @@ const router = express.Router();
...
+
 // GET routes
 router.get('/api/mealInfo/:mealId', mealInfo.get);
+router.get('/api/mealCookInfo/:mealId', cookInfo.get);
```
Comment (DenisKent) [inline server/controllers/index.js]: name as above

**Comment by shiryz** [comment]
@noorafi it's better if you leave a comment on why this pr was closed

**Pull request closed without merging**